engine/src/flutter/display_list/dl_builder.cc

```diff
@@ -1025,6 +1025,42 @@ void DisplayListBuilder::ClipRoundRect(const DlRoundRect& rrect,
       break;
   }
 }
+void DisplayListBuilder::ClipRoundSuperellipse(const DlRoundSuperellipse& rse,
+                                               DlClipOp clip_op,
+                                               bool is_aa) {
+  if (rse.IsRect()) {
+    ClipRect(rse.GetBounds(), clip_op, is_aa);
+    return;
+  }
+  if (rse.IsOval()) {
+    ClipOval(rse.GetBounds(), clip_op, is_aa);
```

Comment:
Is an oval of the RSE bounds really the same as the RSE itself? Doesn't the RSE trace a different curve than an oval?

Reply:
Please ignore, I read further on...